featurebyte/service/session_manager.py

```diff
@@ -32,7 +32,7 @@ def __init__(
     async def get_feature_store_session(
         self,
         feature_store: FeatureStoreModel,
-        get_credential: Any,
+        get_credential: Any = None,
```

Comment:
leaving this parameter here for now since there are a lot of callers. will clean this up in a follow-up.